package-lock.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "postcss-selector-parser",
-  "version": "3.0.0-rc.0",
+  "version": "3.1.1",
   "lockfileVersion": 1,
+  "requires": true,
```

Comment:
I was impressed by the number of changes introduced to the `package.json`. It's introduced with the latest changes made to `npm@5.1.0` (https://github.com/npm/npm/pull/17508).